test/e2e/network/service.go

```diff
@@ -1920,7 +1920,7 @@ var _ = SIGDescribe("Services", func() {
 		err = t.DeleteService(serviceName)
 		framework.ExpectNoError(err, "failed to delete service: %s in namespace: %s", serviceName, ns)
 
-		hostExec := launchHostExecPod(f.ClientSet, f.Namespace.Name, "hostexec")
+		hostExec := e2epod.CreateExecPodOrFail(f.ClientSet, f.Namespace.Name, "hostexec", nil)
```

Comment:
This does not create a pod with host networking, whereas the previous code did. Same for below. What is the intended behavior?

Reply:
@johnbelamaric 
Thanks for your review.
That is a nice point I missed.
If applying this change, HostNetwork test cases become less, it is fine to keep as current code.